src/issuance.rs

```diff
@@ -1271,6 +1347,38 @@ mod tests {
         );
     }
 
+    #[test]
+    fn issue_bundle_cannot_be_signed_with_asset_base_identity_point() {
+        let (rng, isk, _, _, bundle, sighash) =
+            identity_point_asset_base_test_params(10, 20, false);
+
+        assert_eq!(
+            bundle.prepare(sighash).sign(rng, &isk).unwrap_err(),
+            AssetBaseCannotBeIdentityPoint
+        );
+    }
+
+    #[test]
+    fn issue_bundle_verify_fail_asset_base_identity_point() {
+        let (mut rng, isk, ik, _, bundle, sighash) =
+            identity_point_asset_base_test_params(10, 20, false);
+
+        let signed = IssueBundle {
+            ik,
+            actions: bundle.actions,
+            authorization: Signed {
+                signature: isk.sign(&mut rng, &sighash),
+            },
+        };
+
+        let prev_finalized = HashSet::new();
```

Comment:
avoid this line by calling ` HashSet::new()` as a param

Reply:
Done.